assets/scss/globals/_row-block.scss

```diff
@@ -0,0 +1,109 @@
+.row-block {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-areas:
+    "a a a a"
+    "b b b b";
+  grid-gap: 16px;
+
+  font-family: $font-open-sans;
+  padding: 64px 0;
+  margin: 0 16px;
+
+  & > * {
+    max-width: 100%;
+    justify-self: center;
+  }
+
+  .row-block__primary-content  { grid-area: a; }
+  .row-block__secondary-content { grid-area: b; }
+
+  &--reverse {
+    grid-template-areas:
+      "b b b b"
+      "a a a a";
+  }
+}
+
+@include device-is('tablet') {
+  .row-block {
+    margin: 0 32px;
+    grid-template-columns: repeat(8, 1fr);
+    grid-template-areas:
+      'a a a a a a a a'
+      'b b b b b b b b';
+
+    &--reverse {
+      grid-template-areas:
+        'b b b b b b b b'
+        'a a a a a a a a';
+    }
+  }
+}
+
+@include device-is('desktop') {
+  .row-block {
+    margin: 0 32px;
```

Comment:
**suggestion**: il est possible de supprimer ce margin, dans ce cas ce sera le margin de `device-is('tablet')` qui sera pris en compte